roles/sync_monitor/templates/sync_monitor.sh.j2

```diff
@@ -36,25 +35,18 @@ log "Local block height:  $LOCAL_HEIGHT"
 log "Remote block height: $REMOTE_HEIGHT"
 log "Difference: $DIFF blocks"
 
-# Check difference threshold
-if (( DIFF > THRESHOLD )); then
-
-  log "Node is behind by more than $THRESHOLD blocks. Restarting service..."
-
-  systemctl restart zilliqa.service
-  sleep 60
+sleep 90
 
-  # Recheck after restart
-  NEW_LOCAL_HEIGHT=$(get_block_height "$API_URL")
-  NEW_DIFF=$((REMOTE_HEIGHT - NEW_LOCAL_HEIGHT))
+NEW_LOCAL_HEIGHT=$(get_block_height "$API_URL")
+NEW_DIFF=$((REMOTE_HEIGHT - NEW_LOCAL_HEIGHT))
```

Comment:
please use new REMOTE_HEIGHT to calculate the new diff

Reply:
In all cases the sycning should be faster than the block proposal, if its slower, then it would never catch up